composer.json

```diff
@@ -21,19 +21,19 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/framework": "^4.10",
-        "monolog/monolog": "~1.15",
+        "php": "^8.1",
+        "silverstripe/framework": "^5",
+        "monolog/monolog": "^3.2",
         "silverstripe/solr-php-client": "^1.0",
```

Comment:
I take it we are not updating this dependency since `solr-php-client` is a generic PHP library without any Silverstripe CMS dependencies.

Reply:
That's right [no deps](https://github.com/silverstripe/solr-php-client/blob/master/composer.json) - it's a fork on an [abandoned](https://github.com/PTCInc/solr-php-client/issues/9) module that I upgraded to [PHP 8.1 compatibility](https://github.com/silverstripe/solr-php-client/commit/0fd29d11e5f9f43f60ef24e39aac6ad057ee3e46)